packages/webapp/pages/[userId]/index.tsx

```diff
@@ -71,14 +59,6 @@ const ProfilePage = ({
       <Readme user={user} />
       {readingHistory?.userReadingRankHistory && (
         <>
```

Comment:
We should not need the fragments here anymore.

Reply:
Right. I was under the impression that we would not need the `readingHistory?.userReadingRankHistory` anymore. Though after checking, it seems to be used to check whether there is data already to decide whether to render the components.

While the statement about conditionally rendering is true, we might need to use a different variable since the `readingHistory?.userReadingRankHistory` should not exist anymore after the cleanup of the queries.

Reply:
Yup, will spend a bit more on this to possibly get rid of everything that mentions rank